### ISSUE TITLE
add releases

### ISSUE THREAD

**Issue by joseluis** [osnr/TabFS#35]
in order to be able to keep track of updates, without having to follow issues/prs. Thank you

**Comment by jtagcat** [comment]
you may follow commits on master

the situation tfs is in atm… it's very rapidly changing — making releases doesn't make sense for now
If you insist(..?), I guess if we leave this issue open for the future, you'll get pinged when the issue closes.

**Comment by osnr** [comment]
Yeah, I think you raise a good point, and it's definitely worth thinking about lighter ways to make the flow of updates clear (link to the list of commits on master? some kind of running changelog?), so we should leave this open.

but for now, having explicit releases would be extra work & I think would distort development in a way I don't want to do just yet (because we'd start queueing things up in preparation for each release, version numbers would set an expectation of stability that we can't meet, things like that)

**Comment by jtagcat** [comment]
(jfyi, if you didn't know, you can subscribe to gh branch commits on both gh native notifications and rss, etc)
